### DISCUSSION TITLE
replace "Talk to Bot" with "Talk to <name of the chatbot>

### DISCUSSION THREAD

**Discussion by JanKeesSchakel** [langgenius/dify#11893] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I like to create a more personal experience with our Educational WildChats© "Chat with Lupo the Wolf".

See https://www.sensingclues.org/wildchats-lupo

With the help of the Chatbot icon and more, that works really well, up to the point that you enter the chat. There you suddenly do not talk with our wolf anymore, but see the message: "Talk to Bot"

Can an option be added to customise that prompt?

### 3. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

**Comment by wusualis** [comment]
@JanKeesSchakel Hello, May I ask you where to replace "Talk to Bot"? I had replace `web/i18n/en-US/common.ts` in line 546. And then restart the container `docker-web-1`. But it does't work.

**Reply by vvhr** [reply]
You are modifying local code, while your Docker is running an online image. These are two different things If you have developed the code again, you need to build an image and publish it to Docker Hub, and then modify the image source in Docker Composer

**Reply by wusualis** [reply]
Got it. Thanks
